lib/screens/create_region_event_screens/components/review_and_publish_region_event_middle.dart

```diff
@@ -17,32 +16,21 @@ class ReviewAndPublishRegionEventMiddle extends StatelessWidget {
           child: Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
+              Text(state.eventName, style: Theme.of(context).textTheme.headline7),
+              const SizedBox(height: 10),
               Row(
                 children: [
-                  Expanded(child: Text(DateFormat.yMMMMEEEEd().format(state.eventStartDate!))),
-                  Container(
-                    padding: const EdgeInsets.all(6),
-                    decoration: BoxDecoration(
-                      border: Border.all(
-                        color: AppColors.green1,
-                        width: 2,
-                      ),
-                      borderRadius: BorderRadius.circular(12),
-                    ),
-                    child: Row(
-                      children: [
-                        const Icon(Icons.access_time, color: AppColors.green1),
-                        const SizedBox(width: 6),
-                        Text(
-                          DateFormat.jm().format(state.eventStartDate!),
-                          style: Theme.of(context).textTheme.subtitle2Green3LowEmphasis,
-                        ),
-                      ],
-                    ),
-                  )
+                  const Icon(Icons.calendar_today_outlined, color: AppColors.green1, size: 30),
+                  const SizedBox(width: 10),
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Text('Start / ${state.startDateAndTimeFormatted}'),
```

Comment:
Please use 
Text('Start: ${state.startDateAndTimeFormatted}'),

Same for end. :